### DISCUSSION TITLE
Issue number missing from issue and PR titles

### DISCUSSION THREAD

**Discussion by jbrooksuk** [community/community#30044] [Projects and Issues] [answered]
I've noticed today that the issue number has been removed when viewing an issue or pull request. Take for example, https://github.com/laravel/laravel/pull/5969

<img width="1275" alt="CleanShot 2022-08-19 at 09 22 37@2x" src="https://user-images.githubusercontent.com/246103/185576763-3330b793-db34-4326-84d5-f01a03ca277a.png">

The PR number is missing from here and the only place to find it is now in the URL.

The removal of this is strange because we often refer to issues and PRs by their number.

**Comment by mesqueeb** [comment]
I have the same issue!! I need the number back because i'd always copy it together with the title to fill into commits.

Please bring back the issue number...

**Comment by azenMatt** [comment]
Thanks for raising this - we are looking into it

**Comment by hunterliu1003** [comment]
I need this too

**Comment by azenMatt** [comment, marked as answer]
This is fixed. Thanks @jbrooksuk for the spot!

**Reply by jbrooksuk** [reply]
Awesome, thanks folks!

**Reply by stairjoke** [reply]
Great to hear!
